examples/simple-platform/run.adoc

```diff
@@ -0,0 +1,109 @@
+= Run the Example Locally
+
+This section describes how to start all four apps that are used in this
```

Comment:
Can we specifically say Template app, Accounts app, `platform-deploy` node, and `web-auth-endpoint` node since the last two are not apps?